docs/home.md

```diff
@@ -0,0 +1,18 @@
+# Home Page
+
+---
+
+## [Pyalgotrading](pyalgotrad/index.md)
+
+---
+Pyalgotrading is an official Python package that offers algorithmic trading APIs powered by [AlgoBulls](https://www.algobulls.com/). It provides support for creating and uploading strategies on the [AlgoBulls](https://www.algobulls.com/) platform, as well as a pool of free strategies at [pyalgostrategypool](https://github.com/algobulls/pyalgostrategypool). It
+also supports over 150 technical indicators, multiple candlestick patterns and intervals, and different types of orders. Pyalgotrading allows for backtesting, paper trading, and real trading (subject to approval), and supports multiple brokers for
+real trading. Real-time logs and reports are available for all types of trading, and candlestick charts can be plotted using [plotly.py](https://github.com/plotly/plotly.py).
+
+---
+
+## [PythonBuild](python_build/index.md)
+
+---
+The PythonBuild service allows users to code their own trading strategy from scratch or modify templates. Users can access the [AlgoBulls](https://www.algobulls.com/) Core Trading Engine and historical data to gauge strategy performance. Users can tweak parameters and view
```

Comment:
`or modify templates`

need good clarity and coverage on templates. Maybe one or two additional lines can highlight this

Reply:
` and view visual analytics to improve performance`

this should be something along the lines of
` and use the visual analytics to gauge and improve the performance of the strategy`

Reply:
feel free to improve both lines @prithviraj-gotepatil-algobulls 

you can again use generative and/or summarization ai tools if you have done so before

Reply:
done